website/blog/2024-04-22-0.74-yoga-improvements-preparing-for-the-new-architecture.md

```diff
@@ -0,0 +1,441 @@
+---
+title: 'React Native 0.74 - Yoga 3.0, New Architecture Improvements, and more'
+authors: [hurali, alanjhughes, alfonsocj, huntie, cortinico]
+tags: [announcement, release, yoga]
+date: 2024-04-22
+---
+
+Today we're releasing React Native 0.74! This release adds Yoga 3.0, Bridgeless by default under the New Architecture, batched `onLayout` updates, and Yarn 3 as the default package manager for new projects.
+
+We are also actioning API removals for the deprecated `PropTypes` and `PushNotificationIOS` APIs. On Android, SDK 23 (Android 6.0) is now the minimum supported version.
```

Comment:
```suggestion
We are also removing the deprecated `PropTypes` and `PushNotificationIOS` APIs. On Android, SDK 23 (Android 6.0) is now the minimum supported version.
```

Reply:
Will revert this edit, but reword. `PushNotificationIOS` is *not* being removed (yet!).